### PR TITLE
Update StudioReader.cs

### PR DESCRIPTION
**Pull request by SiNeumann**
Use luminance as EmissiveColor in 3ds Importer, if 3ds doesn't contain luminance it falls back to Transparent anyway

### CODE REVIEW

File: Source/HelixToolkit.Wpf/Importers/StudioReader.cs [hunk 1]
```diff
@@ -547,7 +547,7 @@ private void ReadMaterial(BinaryReader reader, int chunkSize)
                         }
 
                         mg.Children.Add(new SpecularMaterial(new SolidColorBrush(specular), specularPower));
-
+                        mg.Children.Add(new EmissiveMaterial((new SolidColorBrush( luminance))));
```
Comment (objorke) [inline Source/HelixToolkit.Wpf/Importers/StudioReader.cs]: why the extra () and space? ;-)

**Comment by objorke** [comment]
Do we have an example where we can see how this looks?

File: Source/HelixToolkit.Wpf/Importers/StudioReader.cs (hunk 1 above)
Comment (objorke) [inline Source/HelixToolkit.Wpf/Importers/StudioReader.cs]: could this line be added only if luminance is not transparent/undefined?

Comment (SiNeumann) [inline Source/HelixToolkit.Wpf/Importers/StudioReader.cs]: Maybe I was to fast with the pull request, I'll add an if clause. Anyway this was just a quirky workaroun, as I use the HelixToolkit.WPF.StudioImport in the Sharp.DX extension. Basically just converting the resulting WPF MeshGeometry3D into the according SharpDX type. So I should add a proper handling for this either by
1. Adding a proper Converter from WPF MeshGeometry3d to SharpDX 
or
2. Port the StudioImporter to HelixToolkit.Wpf.SharpDX

Sorry for the inconvenience.

Comment (objorke) [inline Source/HelixToolkit.Wpf/Importers/StudioReader.cs]: Ok, the functionality of the wpf 3DS importer should not be broken.
The plan is to move the importers and exporters into the portable library #42.
I prefer your alternative 1.

**Pull request closed without merging**